tests/unit/io/WKBReaderTest.cpp

```diff
@@ -52,6 +52,23 @@ struct test_wkbreader_data {
         wktreader(gf.get())
     {}
 
+    void
+    testParseError(const std::string& hexwkb, const std::string& errstr)
+    {
+        std::string exstr;
+        std::stringstream hexin(hexwkb);
+        try {
+            GeomPtr g(wkbreader.readHEX(hexin));
+            ensure("Parse did not fail", !g);
```

Comment:
I think there is a `fail()` or `failure()` or something you can call for this purpose.